src/Microsoft.Framework.CopyOnWriteDictionary.Internal/project.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "1.0.0-*",
+    "shared": "*.cs",
+    "dependencies": {
```

Comment:
curious: why doesn't this project have a build-time dependency on Microsoft.Framework.NotNullAttribute.Internal?